libraries/chain/protocol/transaction.cpp

```diff
@@ -402,9 +407,17 @@ void precomputable_transaction::validate() const
 {
    if( _validated ) return;
    transaction::validate();
+   get_packed_size();
```

Comment:
Opinion: set  _packed_size here by calling transaction::get_packed_size() (slightly more efficient) or call precomputable_transaction::get_packed_size() (slightly more maintainable)?

Reply:
The call doesn't belong here at all. :-) `validate()` is supposed to check the validity of the TX, but you're only calling it for its side effect, i. e. to compute the size.
Better put it in https://github.com/bitshares/bitshares-core/blob/test-2.0.190212/libraries/chain/db_block.cpp#L747

Reply:
Who writes this garbage? Oh, it's me. :-(